packages/flame/lib/src/sprite_animation.dart

```diff
@@ -316,11 +330,16 @@ class SpriteAnimation {
     if (_done) {
       return;
     }
+    if (!_started) {
+      onStart?.call();
+      _started = true;
```

Comment:
```suggestion
      _started = true;
      onFrame?.call(0);
```

Reply:
I fixed it a different way, let me push and take a look.